src/controllers/dynakube/dtclient_reconciler_test.go

```diff
@@ -36,8 +36,6 @@ func TestReconcileDynatraceClient_TokenValidation(t *testing.T) {
 		rec := &DynatraceClientReconciler{
 			Client:              c,
 			DynatraceClientFunc: StaticDynatraceClient(dtcMock),
-			UpdatePaaSToken:     true,
```

Comment:
this could use a test to make sure the `apiToken` is used when the `paasToken` is missing